src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -74,4 +79,29 @@ public PostsListPage clickOnTrendingLinkMobile() {
   public String getSortButtonLabel() {
     return labelInSortEntryPointMobile.getText();
   }
+
+  public void scrollToBottom(WebDriver driver) {
+    ((JavascriptExecutor) driver)
+        .executeScript("window.scrollTo(0, document.body.scrollHeight)");
+  }
+
+  public int getPostsListLength() {
+    wait.forElementNotVisible(By.className(".spinner"));
+    return postList.size();
+  }
+
+  public void clickBackToWikiLink() {
+    backToWiki.click();
+  }
+
+  public void verifyAvatarLoadsUserPage() {
```

Comment:
Don't create `verify` methods, instead use assertions in test classes on returned values from this file i.e.:
```
Assertion.assertTrue(postsList.isUserPageHeaderVisible);
```